apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/BeneficiaryTransformer.java

```diff
@@ -258,92 +258,92 @@ private static Patient transform(Beneficiary beneficiary, RequestHeaders request
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_03,
+                beneficiary.getMedicaidDualEligibilityMarCode()));
+      }
+      if (beneficiary.getMedicaidDualEligibilityAprCode().isPresent()) {
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_04,
+                beneficiary.getMedicaidDualEligibilityAprCode()));
+      }
+      if (beneficiary.getMedicaidDualEligibilityMayCode().isPresent()) {
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_05,
+                beneficiary.getMedicaidDualEligibilityMayCode()));
+      }
+      if (beneficiary.getMedicaidDualEligibilityJunCode().isPresent()) {
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_06,
+                beneficiary.getMedicaidDualEligibilityJunCode()));
+      }
+      if (beneficiary.getMedicaidDualEligibilityJulCode().isPresent()) {
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_07,
+                beneficiary.getMedicaidDualEligibilityJulCode()));
+      }
+      if (beneficiary.getMedicaidDualEligibilityAugCode().isPresent()) {
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_08,
+                beneficiary.getMedicaidDualEligibilityAugCode()));
+      }
+      if (beneficiary.getMedicaidDualEligibilitySeptCode().isPresent()) {
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_09,
+                beneficiary.getMedicaidDualEligibilitySeptCode()));
+      }
+      if (beneficiary.getMedicaidDualEligibilityOctCode().isPresent()) {
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_10,
+                beneficiary.getMedicaidDualEligibilityOctCode()));
+      }
+      if (beneficiary.getMedicaidDualEligibilityNovCode().isPresent()) {
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_11,
+                beneficiary.getMedicaidDualEligibilityNovCode()));
+      }
+      if (beneficiary.getMedicaidDualEligibilityDecCode().isPresent()) {
+        patient.addExtension(
+            TransformerUtils.createExtensionCoding(
+                patient,
+                CcwCodebookVariable.DUAL_12,
+                beneficiary.getMedicaidDualEligibilityDecCode()));
+      }
```

Comment:
here is one spot where the method could still be factored out

Reply:
yes done

Reply:
looks the same to me. am I missing something?

Reply:
fixed
 sorry